coderd/coderdtest/swaggerparser.go

```diff
@@ -300,6 +300,11 @@ func assertPathParametersDefined(t *testing.T, comment SwaggerComment) {
 }
 
 func assertSecurityDefined(t *testing.T, comment SwaggerComment) {
+	authorizedSecurityTags := []string{
+		"CoderSessionToken",
+		"ProvisionerKey",
```

Comment:
What about:

```suggestion
		"CoderProvisionerKey",
```

This would keep the format consistent.

Reply:
Yeah , like it - I just changed and generated doc. ✅